Kong/docs.konghq.com
# Third party plugin cleanup: Set Dynamic Upstream Host plugin

### Description

Removing the Set Dynamic Upstream Host plugin by Flash as part of the third-party plugin cleanup initiative.

### Testing instructions

Netlify link: https://deploy-preview-5451--kongdocs.netlify.app/hub/flash/set-dynamic-upstream-host/ should redirect to /hub/, and the plugin should be gone from the landing page.


### Checklist
